cico_build.sh

```diff
@@ -75,7 +75,12 @@ if [ $? -eq 0 ]; then
     # We are not pushing the nightly tag because we don't need it and CI has an issue
     # when publishing > 1 tag at a time 
     # docker push ${DOCKER_HUB_NAMESPACE}/che-server:${NIGHTLY}
-    docker push ${DOCKER_HUB_NAMESPACE}/che-server:${TAG}
+    export CHE_SERVER_DOCKER_IMAGE=${DOCKER_HUB_NAMESPACE}/che-server:${TAG}
+    cd
+    env_var_file=che.image.env
+    touch $env_var_file
+    echo 'export CHE_SERVER_DOCKER_IMAGE='${CHE_SERVER_DOCKER_IMAGE} > $env_var_file
```

Comment:
I'm afraid that  because of the `cd`, at next loop iteration, commands that depend on path (line 62 and 63) won't work anymore. And these 5 lines can be replaced with one line only:
```sh
echo "export CHE_SERVER_DOCKER_IMAGE=${DOCKER_HUB_NAMESPACE}/che-server:${TAG}" > ~/che.image.env
```